gatsby-browser.js

```diff
@@ -97,8 +96,7 @@ export const onClientEntry = () => {
         device_language: navigator?.language || ' ',
         device_type: isMobile ? 'mobile' : 'desktop',
     })
-    const { tracking } = Analytics.getInstances()
-    tracking.identifyEvent(Analytics?.getId(), { language: getLanguage() })
+    Analytics.identifyEvent()
```

Comment:
Do we need to check if Analytics is available?

Reply:
valid point, let's make it optional 👍